express-api/src/controllers/baseEntity.ts

```diff
@@ -0,0 +1,8 @@
+import { UUID } from 'crypto';
+
+export interface IBaseEntity {
+  createdOn: string;
+  updatedOn: string;
+  createdById: UUID;
+  updatedById: UUID;
+}
```

Comment:
I'm not sure if this is the best place for the baseEntity.ts interface which is the root level of the "controllers" folder.

Reply:
Maybe we could have a `common` folder? For things that apply generally to many controllers.

Reply:
Makes sense.